functional_tests/scylla_operator/test_functional.py

```diff
@@ -356,13 +356,18 @@ def test_scylla_operator_pods(db_cluster: ScyllaPodCluster):
     assert not not_running_pods, f'There are pods in state other than running: {not_running_pods}'
 
 
-def test_startup_probe_in_scylla_pods(db_cluster: ScyllaPodCluster):
-    pods = db_cluster.k8s_cluster.kubectl('get pods -A -l "app.kubernetes.io/name=scylla" -o yaml')
+def test_startup_probe_exists_in_scylla_pods(db_cluster: ScyllaPodCluster):
+    pods = get_pods_without_probe(
```

Comment:
please add argument names that you call in the function

Reply:
Done